pkg/plugins/runtime/k8s/webhooks/injector/injector.go

```diff
@@ -33,6 +33,7 @@ import (
 const (
 	// serviceAccountTokenMountPath is a well-known location where Kubernetes mounts a ServiceAccount token.
 	serviceAccountTokenMountPath = "/var/run/secrets/kubernetes.io/serviceaccount"
+	datadogMountPath             = "/var/run/datadog"
```

Comment:
Does this path is guaranteed by Datadog in every case? Or maybe we should think of adding this into `kuma-cp` config?

Reply:
It's what datadog uses internally, and is used in all their docs. However, this is the point where it is mounted into envoy's container. The user could mount it anywhere in envoy's filesystem, although I don't know why they would choose some other location. Do you think there is a good reason for a user to select a different path?  I can make it configurable if you believe it's necessary.